dpnp/dpnp_array.py

```diff
@@ -292,8 +292,10 @@ def __str__(self):
 
         """
 
-        return str(self.asnumpy())
+        return self._array_obj.__str__()
```

Comment:
need to update an example text above to reflect the new format

Reply:
I do not think the example output actually changed:

```

In [5]: a = dpt.asarray([[136, 136, 136], [272, 272, 272], [408, 408, 408]], dtype='d', device="cpu")

In [6]: print(a)
[[136. 136. 136.]
 [272. 272. 272.]
 [408. 408. 408.]]

In [7]: print(dpt.asnumpy(a))
[[136. 136. 136.]
 [272. 272. 272.]
 [408. 408. 408.]]
```

Reply:
Agree, please ignore my previous comment around that